<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaInsercao</class>
 <widget class="QWidget" name="JanelaInsercao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JanelaInsercao</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../dictionary1600.png</normaloff>../dictionary1600.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inserção</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InsereLinhaPalavra">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite a palavra a inserir.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="InsereMostraTexto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite o significado da palavra.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsereBotaoInserir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsereBotaoVoltar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
